Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -15,6 +15,16 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
 
     public init() {}
 
+    public var flexibleRightSpacing = false
+
+    public init(configuration: AnyObject) throws {
+        flexibleRightSpacing = configuration["flexible_right_spacing"] as? Int == 1
```

Comment:
Hey, Yes I did ...
You asked this already and I answered but I can't seem to find that comment, I guess it got lost. 

You can't directly compare configuration["flexible_right_spacing"] since configuration contains `AnyObject`s (`cannot convert value of type 'AnyObject?!' to expected argument type 'Bool') 

I also rememebr I tried something like this but YAML's `true` didn't convert to Swift `Bool` and this was always false-y.  

`flexibleRightSpacing = configuration["flexible_right_spacing"] as? Bool == true`